src/modules/mavlink/mavlink_main.cpp

```diff
@@ -1074,6 +1074,7 @@ Mavlink::send_autopilot_capabilities()
 		msg.capabilities |= MAV_PROTOCOL_CAPABILITY_MAVLINK2;
 		msg.capabilities |= MAV_PROTOCOL_CAPABILITY_MISSION_FENCE;
 		msg.capabilities |= MAV_PROTOCOL_CAPABILITY_MISSION_RALLY;
+		msg.capabilities |= MAV_PROTOCOL_CAPABILITY_COMPONENT_IMPLEMENTS_GIMBAL_MANAGER;
```

Comment:
@julianoes This is the simplest possible update, which just sets the bit.

It isn't clear to me if there is any way we can check if there is a gimbal manager actually in action and set this conditionally. I am kind of hoping you will take this over and do this the changes if needed. It would also be good to push this change into v1.15, even if we don't do a more precise fix in main.

The [gimbal setup docs](https://docs.px4.io/main/en/advanced/gimbal_control.html#mavlink-gimbal-mnt-mode-out-mavlink) indicate that only one gimbal manager is supported, and that it can support either a mavlink or PWM backend.
The text looks like all we need to do to accept gimbal manager input is set [MNT_MODE_IN](https://docs.px4.io/main/en/advanced_config/parameter_reference.html#MNT_MODE_IN) to `MAVlink gimbal protocol v2`, so perhaps a check would be to read this parameter and check the value?

Reply:
Oh, I see. Good point. We should only set the bit if the param [MNT_MODE_IN](https://docs.px4.io/main/en/advanced_config/parameter_reference.html#MNT_MODE_IN) is set to 4 (gimbal v2).